### PR TITLE
[expo-updates] Rewrite README.md

### PR DESCRIPTION
**Pull request by douglowder**
- Remove stale text from old versions of the module
- Add documentation links

### CODE REVIEW

File: packages/expo-updates/README.md [hunk 1]
```diff
@@ -7,298 +7,22 @@
   </a>
 </p>
 
-`expo-updates` fetches and manages updates to your app stored on a remote server.
+The `expo-updates` module allows your app to download and manage updates to your application code that are stored on a remote server.
```
Comment (wschurman) [inline packages/expo-updates/README.md:10]:
```suggestion
The `expo-updates` module allows your app to download and manage remote updates to your application code.
```

**Review by wschurman — APPROVED** [review]

File: packages/expo-updates/README.md [hunk 2]
```diff
@@ -7,298 +7,22 @@
...
-
-If you're upgrading from `expo-updates@0.1.x`, you can opt into the **no-publish workflow**. In this workflow, release builds of both iOS and Android apps will create and embed a new update at build-time from the JS code currently on disk, rather than embedding a copy of the most recently published update. For instructions and more information, see the [CHANGELOG](https://github.com/expo/expo/blob/main/packages/expo-updates/CHANGELOG.md). (For new projects, the no-publish workflow is enabled by default.)
-
-# Installation in managed Expo projects
-
-For [managed](https://docs.expo.dev/archive/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/updates/).
+- [Getting started](https://docs.expo.dev/eas-update/getting-started/)
+- [How EAS Update works](https://docs.expo.dev/eas-update/how-it-works/)
+- [Debugging](https://docs.expo.dev/eas-update/debug/)
+- [Migration from classic updates](https://docs.expo.dev/eas-update/migrate-from-classic-updates/)
```
Comment (quinlanj) [inline packages/expo-updates/README.md:23]:
```suggestion
- [Migration from Classic Updates](https://docs.expo.dev/eas-update/migrate-from-classic-updates/)
```

**Review by quinlanj — APPROVED** [review]

**Comment by douglowder** [comment]
@amandeepmittal @kbrandwijk I'm going to go ahead and merge this, but if you see changes you want to make, let me know and we can do another PR.

**Comment by amandeepmittal** [comment]
Thank you, @douglowder, I will take a look and let you know.

**Pull request merged**